### PR TITLE
update several statements

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: sql-statements/sql-statement-set-transaction.md [hunk 1]
```diff
@@ -94,7 +94,8 @@ SHOW SESSION VARIABLES like 'transaction_isolation';
 
 * TiDB 支持仅在语法中将事务设置为只读的功能。
 * 不支持隔离级别 `READ-UNCOMMITTED` 和 `SERIALIZABLE`。
-* 隔离级别 `REPEATABLE-READ` 在技术上属于快照隔离（Snapshot Isolation）。在 TiDB 中称为 `REPEATABLE-READ` 是为了和 MySQL 保持一致。
+* 隔离级别 `REPEATABLE-READ` 在乐观事务中实现为快照隔离（Snapshot Isolation），在 TiDB 中称为 `REPEATABLE-READ` 是为了和 MySQL 保持一致。
```
Comment (scsldb) [inline sql-statements/sql-statement-set-transaction.md:97]: 通过快照隔离（Snapshot Isolation）技术实现乐观事务的 `REPEATABLE-READ`  隔离级别，行为和 MySQL 保持一致。

Comment (jackysp) [inline sql-statements/sql-statement-set-transaction.md:97]:
```suggestion
* 通过快照隔离（Snapshot Isolation）技术实现乐观事务的 `REPEATABLE-READ` 隔离级别，和 MySQL 兼容。
```

**Comment by jackysp** [comment]
PTAL @scsldb

File: sql-statements/sql-statement-set-transaction.md [hunk 2]
```diff
@@ -94,7 +94,8 @@ SHOW SESSION VARIABLES like 'transaction_isolation';
 
 * TiDB 支持仅在语法中将事务设置为只读的功能。
 * 不支持隔离级别 `READ-UNCOMMITTED` 和 `SERIALIZABLE`。
-* 隔离级别 `REPEATABLE-READ` 在技术上属于快照隔离（Snapshot Isolation）。在 TiDB 中称为 `REPEATABLE-READ` 是为了和 MySQL 保持一致。
+* 通过快照隔离（Snapshot Isolation）技术实现乐观事务的 `REPEATABLE-READ` 隔离级别，和 MySQL 兼容。
```
Comment (lilin90) [inline sql-statements/sql-statement-set-transaction.md:97]:
```suggestion
* 通过快照隔离 (Snapshot Isolation) 技术，实现乐观事务的 `REPEATABLE-READ` 隔离级别，和 MySQL 兼容。
```

File: sql-statements/sql-statement-set-transaction.md [hunk 3]
```diff
@@ -94,7 +94,8 @@ SHOW SESSION VARIABLES like 'transaction_isolation';
...
+在悲观事务中，TiDB 支持与 MySQL 兼容的 `REPEATABLE-READ` 和 `READ-COMMITTED` 两种隔离级别。具体描述详见 [Isolation Levels](/transaction-isolation-levels.md)。
```
Comment (lilin90) [inline sql-statements/sql-statement-set-transaction.md:98]: 请确认下这行是否也是一个列表项？
```suggestion
* 在悲观事务中，TiDB 支持与 MySQL 兼容的 `REPEATABLE-READ` 和 `READ-COMMITTED` 两种隔离级别。具体描述详见 [Isolation Levels](/transaction-isolation-levels.md)。
```

File: sql-statements/sql-statement-set-transaction.md (hunk 2 above)
Comment (jackysp) [reply sql-statements/sql-statement-set-transaction.md:97]: 可以。

File: sql-statements/sql-statement-set-transaction.md (hunk 3 above)
Comment (jackysp) [reply sql-statements/sql-statement-set-transaction.md:98]: 可以

**Comment by jackysp** [comment]
PTAL @lilin90

**Comment by coocood** [comment]
LGTM

**Comment by jackysp** [comment]
Conflict resolved, @zimulala @coocood @lysu . Need to be approved.

**Pull request merged**